### DISCUSSION TITLE
HTML Files

### DISCUSSION THREAD

**Discussion by MrBrain295** [mdn/mdn-community#99] [Content] [answered]
@Rumyra @estelle @wbamberg
./files/en-us/web/accessibility/implementing_msaa_server/index.html
./files/en-us/web/accessibility/html_to_msaa/index.html
./files/en-us/web/accessibility/aria/aria_test_cases/index.html
./files/en-us/web/accessibility/aria/forms/multipart_labels/index.html
./files/en-us/web/accessibility/aria/forms/basic_form_hints/index.html
./files/en-us/web/accessibility/aria/forms/alerts/index.html
./files/en-us/web/accessibility/aria/web_applications_and_aria_faq/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_presentation_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/x-ms-aria-flowfrom/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_toolbar_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_radio_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_status_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_group_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_link_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_log_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_alertdialog_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_slider_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_progressbar_role/index.html
./files/en-us/web/accessibility/aria/aria_techniques/using_the_article_role/index.html
./files/en-us/web/accessibility/aria/widgets/index.html
./files/en-us/web/accessibility/understanding_wcag/perceivable/color_contrast/index.html
./files/en-us/web/accessibility/understanding_wcag/perceivable/use_of_color/index.html
Migrated from mdn/content#14937.

**Comment by estelle** [comment]
of the above, all should be deleted except for 
./files/en-us/web/accessibility/understanding_wcag/perceivable/color_contrast/index.html
./files/en-us/web/accessibility/understanding_wcag/perceivable/use_of_color/index.html
the role pages have already been deleted / redirected

**Comment by MrBrain295** [comment]
Here is the updated list:
[files/en-us/web/accessibility/aria/web_applications_and_aria_faq/index.html](https://github.com/mdn/content/blob/e6bceecd3a3aa6d6d7223bc11e6254440eb80557/files/en-us/web/accessibility/aria/web_applications_and_aria_faq/index.html)
[files/en-us/web/accessibility/implementing_msaa_server/index.html](https://github.com/mdn/content/blob/0df41fd3125fba4a65e3900e0a6effb7072144c9/files/en-us/web/accessibility/implementing_msaa_server/index.html)
[files/en-us/web/accessibility/understanding_wcag/perceivable/use_of_color/index.html](https://github.com/mdn/content/blob/6d890b9e01cf7b8511a147387e013fee8b072e0a/files/en-us/web/accessibility/understanding_wcag/perceivable/use_of_color/index.html)
[files/en-us/web/accessibility/aria/aria_test_cases/index.html](https://github.com/mdn/content/blob/b3ce3da3b2f83ca40fcc4bdb4f5b11965d5425eb/files/en-us/web/accessibility/aria/aria_test_cases/index.html)
[files/en-us/web/accessibility/aria/forms/multipart_labels/index.html](https://github.com/mdn/content/blob/e1b15977448f3cf7d7f06dc6846574bf4dd7a4a8/files/en-us/web/accessibility/aria/forms/multipart_labels/index.html)
[files/en-us/web/accessibility/aria/aria_techniques/x-ms-aria-flowfrom/index.html](https://github.com/mdn/content/blob/0440b0fa1c116be244d5047c37a0f1197af13282/files/en-us/web/accessibility/aria/aria_techniques/x-ms-aria-flowfrom/index.html)
[files/en-us/web/accessibility/understanding_wcag/perceivable/color_contrast/index.html](https://github.com/mdn/content/blob/0d082b64460e86283136f07650db8a274d240317/files/en-us/web/accessibility/understanding_wcag/perceivable/color_contrast/index.html)
[files/en-us/web/accessibility/aria/forms/alerts/index.html](https://github.com/mdn/content/blob/426485595b1be50e2af91764fbfd94c04c3c36a5/files/en-us/web/accessibility/aria/forms/alerts/index.html)
[files/en-us/web/accessibility/html_to_msaa/index.html](https://github.com/mdn/content/blob/303a6c3b8ec6658d2956c191c50aee0a0e9e4da5/files/en-us/web/accessibility/html_to_msaa/index.html)
[files/en-us/web/accessibility/aria/forms/basic_form_hints/index.html](https://github.com/mdn/content/blob/303a6c3b8ec6658d2956c191c50aee0a0e9e4da5/files/en-us/web/accessibility/aria/forms/basic_form_hints/index.html)
[files/en-us/web/accessibility/aria/widgets/index.html](https://github.com/mdn/content/blob/303a6c3b8ec6658d2956c191c50aee0a0e9e4da5/files/en-us/web/accessibility/aria/widgets/index.html)

**Reply by Rumyra** [reply]
@estelle can you confirm your above comment, that all these should be removed apart from the two you mentioned?

**Reply by estelle** [reply]
i deleted some: https://github.com/mdn/content/pull/19039/. The rest I tried to convert, but while my yari seemed to work, I can't find where the new files were saved.

> yarn md h2m Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast/
> yarn md h2m Web/Accessibility/Understanding_WCAG/Perceivable/Use_of_color/
> yarn md h2m Web/Accessibility/ARIA/forms/alerts/
> yarn md h2m Web/Accessibility/ARIA/forms/Multipart_labels/

The first two need to be converted. The latter two have a little bit of useful information that can be folded into other pages, so I didn't delete them yet.

**Comment by Rumyra** [comment, marked as answer]
Thanks to @queengooborg these final html files have been converted 🎉  - thanks both for the lists!
